lib/OpenQA/CacheService/Model/Cache.pm

```diff
@@ -247,18 +224,12 @@ sub purge_asset {
     my ($self, $asset) = @_;
 
     my $log = $self->log;
-    eval {
-        my $db = $self->sqlite->db;
-        my $tx = $db->begin('exclusive');
-        $db->delete('assets', {filename => $asset});
-        $tx->commit;
-        if (-e $asset) { $log->error(qq{Unlinking "$asset" failed: $!}) unless unlink $asset }
-        else           { $log->debug(qq{Purging "$asset" failed because the asset did not exist}) }
-    };
-    if (my $err = $@) {
-        $log->error(qq{Purging "$asset" failed: $err});
-        return undef;
-    }
+    my $db  = $self->sqlite->db;
+    my $tx  = $db->begin('exclusive');
+    $db->delete('assets', {filename => $asset});
+    $tx->commit;
+    if (-e $asset) { $log->error(qq{Unlinking "$asset" failed: $!}) unless unlink $asset }
+    else           { $log->debug(qq{Purging "$asset" failed because the asset did not exist}) }
```

Comment:
I can't believe **tidy** allows this. This looks very confusing to me

Reply:
this is what tidy is doing itself, it's aligning the curly braces when the if/else was inlined. I did not actually see this or change it, I just changed indendation after removing the surrounding eval.